<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Airlines manager</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <widget class="QTableWidget" name="table_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>590</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>HUB</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TAG</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CATEGORY</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DISTANCE</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_add_info_from_html"/>
    <addaction name="action_save_info_to_am"/>
    <addaction name="action_load_info_from_am"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <action name="action_add_info_from_html">
   <property name="text">
    <string>Add info from .html</string>
   </property>
  </action>
  <action name="action_save_info_to_am">
   <property name="text">
    <string>Save info to .am</string>
   </property>
  </action>
  <action name="action_load_info_from_am">
   <property name="text">
    <string>Load info from .am</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
